src/piwardrive/core/utils.py

```
@@ -0,0 +1,1135 @@
+"""Utility functions for the PiWardrive GUI application."""
+
+# pylint: disable=broad-exception-caught,unspecified-encoding,subprocess-run-check
+
+from . import fastjson
```

Pylint (3.3.7)

[error] No name 'fastjson' in module 'piwardrive.core'

(E0611)